test/controllers/translates_controller_test.rb

```diff
@@ -3,11 +3,17 @@
 require 'test_helper'
 
 class TranslatesControllerTest < ActionDispatch::IntegrationTest
-  test 'post transformation of fgdc to iso19115_3' do
```

Comment:
why delete this test?

Reply:
that test ended up as 'post invalid transformation of fgdc to iso19115_3'. since we're changing the status codes of the API responses that test is no longer valid. also, that original test wasn't working as intended anyway. i wasn't joining the file path of the fixture correctly. this has been fixed in this PR

Reply:
this is a good example of why we're changing the status code. despite sending the wrong data to the API it gave us a 200 back albeit the response does have `success=false`.